src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -191,11 +193,11 @@ function prepareMCQQuestions() {
     var radioButtons = {};
     var radioStates = {};
 
-    for (var i = 0; i < mcqQuestionNums.length; i++) {
+    $.each(mcqQuestionNums, function(i) {
         var qnNum = mcqQuestionNums[i];
         var numResponses = $('[name="questionresponsetotal-' + qnNum + '"]').val();
 
-        for (var j = 0; j < numResponses; j++) {
+        $.each(numResponses, function(j) {
```

Comment:
There are issues with using `$.each(new Array(numResponses), function(j) {` here.
Since there are no function declarations or statements in here (only expressions), I'll just revert to the normal for loop.